spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/EmptyCassandraRDD.scala

```diff
@@ -0,0 +1,45 @@
+package com.datastax.spark.connector.rdd
+
+import com.datastax.spark.connector._
+import com.datastax.spark.connector.cql.CassandraConnector
+import org.apache.spark.annotation.DeveloperApi
+import org.apache.spark.{Partition, SparkContext, TaskContext}
+
+import scala.reflect.ClassTag
+
+class EmptyCassandraRDD[R](@transient sc: SparkContext,
+                           val keyspaceName: String,
+                           val tableName: String,
+                           val columnNames: ColumnSelector = AllColumns,
+                           val where: CqlWhereClause = CqlWhereClause.empty,
+                           val limit: Option[Long] = None,
+                           val clusteringOrder: Option[ClusteringOrder] = None,
+                           val readConf: ReadConf = ReadConf())
+                          (implicit val rct: ClassTag[R])
+  extends CassandraRDD[R](sc, Seq.empty) {
+  override protected def copy(columnNames: ColumnSelector = columnNames,
```

Comment:
Cosmetics: Can you move this line a little lower? Visually it looks as if it belonged to the class header.